LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -58,7 +58,7 @@ public class LabClient implements ILabClient {
 
     private final LabApiAuthenticationClient mLabApiAuthenticationClient;
     private final long PASSWORD_RESET_WAIT_DURATION = TimeUnit.SECONDS.toMillis(65);
-    private final long LAB_API_RETRY_WAIT = TimeUnit.SECONDS.toMillis(2);
+    private final long LAB_API_RETRY_WAIT = TimeUnit.SECONDS.toMillis(8);
```

Comment:
It would be good to document the reasons for these changes as part of the PR description

Reply:
This was a leftover change, new api was having some issues consistently making the temp users. I'll add a comment

Reply:
Removed this, not needed.